src/Microsoft.ML.FastTree/RegressionTree.cs

```diff
@@ -56,7 +56,9 @@ public abstract class RegressionTreeBase
         /// Note that the case (1) happens only when <see cref="CategoricalSplitFlags"/>[i] is true and otherwise (2)
         /// occurs. A non-negative returned value means a node (i.e., not a leaf); for example, 2 means the 3rd node in
         /// the underlying <see cref="_tree"/>. A negative returned value means a leaf; for example, -1 stands for the
-        /// first leaf in the underlying <see cref="_tree"/>.
+        /// <see langword="~"/>(-1)-th leaf in the underlying <see cref="_tree"/>. Note that <see langword="~"/> is the
+        /// bitwise complement operator in C#; for details, see
+        /// https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/bitwise-complement-operator.
```

Comment:
see <a href="[https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/bitwise-complement-operator](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/bitwise-complement-operator)">the documentation about the bitwise complement operator </a>